include/flatbuffers/flatbuffers.h

```diff
@@ -69,7 +69,7 @@ template<typename T> T EndianSwap(T t) {
   }
 }
 
-template<typename T> size_t AlignOf() {
+template<typename T> constexpr size_t AlignOf() {
```

Comment:
what is this needed for? there's no `constexpr` anywhere else in the code, so I'd rather not have it for the sake of older compilers, for the moment.